addOns/automation/src/main/javahelp/org/zaproxy/addon/automation/resources/help/contents/job-ascan.html

```diff
@@ -44,6 +44,8 @@ <H2>YAML</H2>
         threshold:                     # String: The Alert Threshold for this rule, one of Off, Low, Medium, High, default: Medium
 </pre>
 
+<strong>Note</strong>: Unless the <code>defaultThreshold</code> of the <policyDefinition> is <code>OFF</code> all rules will be enabled to start with.
```

Comment:
The `policyDefinition` is missing the `code` tags (or remove the `<>`), same for other help page.